### PR TITLE
Render Users, Posts and Comments Views on the App UI

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
In this PR I added the views associated with the Controllers and Models in the app.

**Below are the highlights**
Create the view for all the users
Create the view for a specific user's posts
Create the view for a specific post's comments

**Cheers and Happy Reviewing!**

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,30 @@
...
+<p><%= @user.name %></p>
+<p>Number of Posts: <%= @user.posts_counter %></p>
+</div>
+<% @user.posts.each do |post| %>
+ <div>
+ <p><%= link_to user_post_path(@user, post.id) do %></p>
+ <p>Post <%= post.id  %></p>
+ <% end %>
+ <p><%= post.text  %></p>
+ <p>Comments <%= post.comments_counter  %></p>
```
Comment (rloterh) [inline app/views/posts/index.html.erb]: - [ ] `@post.most_recent_comments` is missing in your post _index_ view. This activity requires that we use the `recent_comments` method defined in the _post_ model, in your case `most_recent_comments`, to render the user's 5 most recent comments in the `app/views/posts/index.html.erb` view, it should be similar to:

```ruby
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

Kindly ensure to improve upon the same.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    <div>
-    <p><%= user.bio %></p>
+    <img src= "<%= user.photo %>" />
     </div>
     <div>
-    <p><%= user.photo %></p>
+    <%= link_to user_path(user.id) do %>
+    <p><%= user.name %></p>
+    <% end %>
+    <p>Number of Posts: <%= user.posts_counter %></p>
```
Comment (rloterh) [inline app/views/users/index.html.erb]: - [ ] Kindly ensure that the users page matches the given [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- [ ] Navigating to `http://[::1]:3000/users/1/posts/1/` to get post of the first user is not rendering correctly. Kindly ensure to improve upon the same.

**Your Users view:**
image

**Users view (wireframe):**
image

**Users post view error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Sharon, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Good job with views implementation so far
- Good ReadMe

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Comments controller is missing, and hence, unable to generate sample data for comments. Kindly endeavor to add a comments controller with a create action.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sharon-odhiambo** [comment]
Hi @rloterh , thank you for the review. Are you available for some clarifications?

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @sharon-odhiambo 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, not criticizing it. 😄

 ## Required Changes ♻️

- [ ] Please make sure to add styles like the given wireframe as the previous reviewer requested. 🙏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rloterh** [comment]
Hi @sharon-odhiambo
I'm available. I just saw your message via slack. 😃

**Comment by sharon-odhiambo** [comment]
Hi @MahmoudBakr23 , thank you for the review. I've made the required changes.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @sharon-odhiambo👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sharon-odhiambo** [comment]
Hi @thecodechaser , thank you for the review.

**Pull request merged**